runners/gradle-plugin/src/main/kotlin/org/jetbrains/dokka/gradle/ConfigurationExtractor.kt

```diff
@@ -24,13 +24,13 @@ import java.io.Serializable
 
 class ConfigurationExtractor(private val project: Project) {
 
-    fun extractConfiguration(targetName: String, variantName: String?) = if (project.isMultiplatformProject()) {
-        extractFromMultiPlatform(targetName, variantName)
+    fun extractConfiguration(targetName: String, variantNames: List<String?>) = if (project.isMultiplatformProject()) {
```

Comment:
I think we don't need `List<String?>` here, `List<String>` would be sufficient, because the nullability here doesn't have any meaning